cmd/swarmctl/main.go

```diff
@@ -41,7 +41,7 @@ func defaultSocket() string {
 	if swarmSocket != "" {
 		return swarmSocket
 	}
-	return "./swarmkitstate/swarmd.sock"
+	return "/run/swarmd/swarmd.sock"
```

Comment:
Docker's socket is `/var/run/docker.sock` by default (IIRC). Shouldn't we be consistent with that?